### ISSUE TITLE
Links in level 1 headings are hard to read with the default (dark?) palette

### ISSUE THREAD

**Issue by spenserblack** [charmbracelet/glamour#165]
[I originally raised this issue at cli/cli](https://github.com/cli/cli/issues/6351), but since they use glamour and don't customize the color palette it was recommended to raise the issue here.

******

When a level 1 heading contains a link, the purple-ish background and the dark cyan foreground of the link result in a low contrast that makes the text hard to read.

Here is a screenshot: https://github.com/cli/cli/issues/6351#issuecomment-1258321751

**Comment by muesli** [comment]
Agreed, this doesn't look nice. We should probably skip the link styling within headings.

**Comment by brendencallahan** [comment]
I just opened an issue in the glow repo (which can be seen above this comment) related to this. 

@meowgorithm suggested using `123` for the foreground of links. I think that's a great idea, but looking at the screenshot it seems that the background clashes with the text before the URL as well. Maybe the background color should be changed instead?

In that issue I also suggested fixing it in the glow repo but this seems like a better place, since it is the root cause. If it's okay, I would like to fix this. Let me know what colors you decide on.

**Comment by brandon-devs** [comment]
Is this problem specific to the dark theme?

**Comment by brendencallahan** [comment]
Actually, I think it is both the dark and light theme. I've attached screenshots below. I might suggest doing something similar to the Dracula theme and possibly not having a background color. I was planning on fixing this after a color was decided on. 

![Screenshot_2023-01-28-10-47-41_1920x1080](https://user-images.githubusercontent.com/47364240/215285553-a18fe991-0aec-496b-a85d-6141326dd401.png)
![Screenshot_2023-01-28-10-47-21_1920x1080](https://user-images.githubusercontent.com/47364240/215285557-0c6af704-77cd-4517-ba8b-2a080b7dbd95.png)

**Comment by brandon-devs** [comment]
@brendencallahan Alright, thanks. I made a PR but I’ll edit it to include the other themes. Also, what terminal / shell theme are you using? Looks really nice :-)

**Comment by brendencallahan** [comment]
@brandon-devs I see your pr now. Nice! Terminal is kitty with 0.9 opacity and #2d2c35 background Jetbrains mono font. Shell is zsh with powerlevel10k theme.
